tools/wptrunner/wptrunner/executors/executorchrome.py

```diff
@@ -93,53 +93,41 @@ def setup(self):
         self.test_window = None
```

Comment:
Can we move `test_window` to `WebDriverTestharnessProtocolPart`  but let `reuse_window` always be false there, such that we functions, e.g. `close_old_windows`, can be kept the same between the two?